setup_script

```diff
@@ -58,7 +58,9 @@ echo "Setting up MongoDB"
 sudo mkdir -p /data/db/
 sudo chown -R `id -un` /data/db/
 sudo service mongodb start
+cd ./Database\ Structures
```

Comment:
This no longer needs a space in between the names. Can you fast-forward and recommit with this change? I just approved the pull_request with those changes